brave/brave-core
# fix(wallet): Duplicate Tokens on Fund Wallet Page

## Description 
`WBTC`'s  contract address was a `Wallet` address `0x1bfd67037b42cf73acf2047067bd4f2c47d9bfd6`
- updated to `0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599` https://etherscan.io/token/0x2260fac5e5542a773aa44fbcfedf7c193bc2c599?a=0x1bfd67037b42cf73acf2047067bd4f2c47d9bfd6


`BAT` on `Solana` was using an `Ethereum` contract address `0x0D8775F648430679A709E98d2b0Cb6250d2887EF`
- updated to `EPeUFDgHRxs9xxEPVaL6kfGQvCon7jmAWKVUHuux1Tpz` https://explorer.solana.com/address/EPeUFDgHRxs9xxEPVaL6kfGQvCon7jmAWKVUHuux1Tpz


Updated `Compound` tokens to have capital `COMP` for the symbols.



Resolves <https://github.com/brave/brave-browser/issues/27617>

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

   1. Go to the `Fund Wallet` page
   2. Search for `WBTC`, there should not be duplicate listings for the `Ethereum` network
   3. Search for `BAT`, there should not be duplicate listings for the `Solana` network

https://user-images.githubusercontent.com/40611140/210427454-4f5c0084-aefd-4841-a06b-cb649b68571f.mov
